grammarly.tex

```diff
@@ -1,4 +0,0 @@
-\def\grammarly{on}
```

Comment:
Should we remove it from the `Makefile` as well then?

Reply:
This was already removed in 97cc4b747d7160c7585cd651baf7cb7d6a96da09/

Reply:
Oh okay, then that's good!